### DISCUSSION TITLE
Transaction specific purpose in OpenID Connect for Identity Assurance 1.0 spec

### DISCUSSION THREAD

**Discussion by shilpa-padgaonkar** [camaraproject/IdentityAndConsentManagement#39] [General]
The specs which includes details about transaction specific purpose can be found here:
https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html 

Bjorn Hjelm from OIDF has confirmed that the above link is the latest.
He has provided this additional link  https://openid.net/wg/ekyc-ida/charter/ where we can find more background info on the topic.

He can help arrange a dedicated session in the subproject when we have questions on the topic.

**Comment by jpengar** [comment]
@shilpa-padgaonkar Is this the information we requested by email in the CAMARA Commonalities WG mail list? (Email subject: `Re: [sp-icm] Discussion between CAMARA and OpenID Foundation`). Or is this coming from another track or something?

We will take a look at it.

**Reply by bhjelm** [reply]
The [OpenID Connect for Identity Assurance](https://openid.net/specs/openid-connect-4-identity-assurance-1_0.html) specification defines the technical mechanisms to (1) allow RPs to request Verified Claims (about an End-User) and (2) enable OPs to provide RPs with Verified Claims. This may (or may not) be more applicable to the [KnowYourCustomer SP](https://camaraproject.org/knowyourcustomer/) group. However, it's a topic that is part of the liaison discussion between the OpenID Foundation and CAMARA.
